regular-expressions/introduction/basic-syntax-i/regex-wildcards-i.md

```diff
@@ -64,9 +64,9 @@ Which regex pattern would match the following string?
 
 ???
 
-* `/\w+/`
-* `/\W+/`
-* `/./`
+* `/\w/`
```

Comment:
`/\w+/` was the correct answer.

`/\w` will match `1`, `2`, `3` but it won't match `123`

Reply:
ah yep, the users comment made me worry about full matches so I took these `+` out. I will add them back in though!